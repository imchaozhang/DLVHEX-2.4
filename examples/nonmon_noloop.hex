% this program tests the basic functionality of the
% nonmonotonic external atom testSetMinus
seta(u). seta(v). seta(w).
setb(u).

onlyInA(X) :- &testSetMinus[seta,setb](X).
