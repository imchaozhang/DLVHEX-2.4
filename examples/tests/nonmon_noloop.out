{onlyInA(w), onlyInA(v), setb(u), seta(w), seta(v), seta(u)}
